README.adoc

```diff
@@ -3604,6 +3604,21 @@ end
 Person = Struct.new(:first_name, :last_name)
 ----
 
+=== Don't Extend `Data.define` [[no-extend-data-define]]
+
+Don't extend an instance initialized by `Data.define`.
+Extending it introduces a superfluous class level.
+
+[source,ruby]
+----
+# bad
+class Person < Data.define(:first_name, :last_name)
+end
```

Comment:
I think it will also be a good idea to show the output of `Person.ancestors` in both examples to show clearly the anonymous class that gets added there.

Reply:
Done! Do you think I should open another one to illustrate this for `[[no-extend-struct-new]]` too?